x/feegrant/keeper/msg_server.go

```diff
@@ -84,3 +85,22 @@ func (k msgServer) RevokeAllowance(ctx context.Context, msg *feegrant.MsgRevokeA
 
 	return &feegrant.MsgRevokeAllowanceResponse{}, nil
 }
+
+// PruneAllowances removes expired allowances from the store.
+func (k msgServer) PruneAllowances(ctx context.Context, req *feegrant.MsgPruneAllowances) (*feegrant.MsgPruneAllowancesResponse, error) {
+	// 75 is an arbitrary value, we can change it later if needed
+	err := k.RemoveExpiredAllowances(ctx, 100)
```

Comment:
```suggestion
	err := k.RemoveExpiredAllowances(ctx, 75)
```
shouldn't this be 75 ?

Reply:
yes, thank you for spotting it! might have committed while testing